ready.go

```diff
@@ -129,12 +129,22 @@ func newConfig() *config {
 }
 
 func (c *config) withYAML() (*config, error) {
-	path, err := exec.Command("pwd").CombinedOutput()
+	var path []byte
+	var err error
+
+	if runtime.GOOS == "windows" {
+		dir, errDir := os.Getwd()
+		err = errDir
+		byteDir := []byte(dir)
+		path = byteDir
+	} else {
+		path, err = exec.Command("pwd").CombinedOutput()
+	}
```

Comment:
Thanks for this! We can even simplify it as:

```suggestion
	path, err := os.Getwd()
```

As this method will take care of using the correct command depending on the operating system: https://pkg.go.dev/os#Getwd